script/release/notes/notes.js

```diff
@@ -130,6 +130,8 @@ const getNoteFromClerk = async (ghKey) => {
         .trim();
     }
   }
+
+  console.log(`WARN: no notes found in ${buildPullURL(ghKey)}`);
```

Comment:
nit, but since it does change display and parsing a little bit, we should consider using `console.warn`

Reply:
I like that! Updated in 62cbc5fddb035ed9db9b646a61268fcbf81d2d1b